src/System.Text.Json/tests/Serialization/Value.ReadTests.cs

```diff
@@ -3,13 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Globalization;
-using System.Runtime.CompilerServices;
 using Xunit;
 
 namespace System.Text.Json.Serialization.Tests
 {
     public static partial class ValueTests
     {
+        public const int MaxEscapedTokenSize = 1_000_000_000;   // Max size for already escaped value.
+        public const int MaxExpansionFactorWhileTranscoding = 3;
+
+        public static bool IsX64 { get; } = IntPtr.Size >= 8;
```

Comment:
```suggestion
        public static readonly bool IsX64 { get; } = Environment.Is64BitProcess;
```
Actually does the same, but is more verbose.

Reply:
Thanks changing to
`public static readonly bool IsX64 = Environment.Is64BitProcess;`

There are a couple other similar locations in System.Text.Json that I did not update here to keep this PR lightweight for potential merge into 3.1.